### DISCUSSION TITLE
플레이어가 Room에 입장했는데 room 객체가 null일 수 있나요?

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1072] [5. Q&A]
![image](https://user-images.githubusercontent.com/59792911/180937691-18f5e623-8506-429e-8053-59d90dbf9d05.png)
플레이어 입장 이후 룸 객체를 받아서 this.RoomSendMessage를 하려고 합니다.
그런데 this.room이 null이라 할 수 없다고 하네요.
multiplay example에서 제공하는 코드와 거의 유사합니다.

![image](https://user-images.githubusercontent.com/59792911/180937900-d62ff8c6-3af3-4da6-b78a-0de2d5cd2ded.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 코드 로직 오류는 보이지 않습니다.
테스트 당시 멀티플레이 서버는 켜져있었는지 확인해보면 좋을 것 같습니다.
